### ISSUE TITLE
Many to many bindings

### ISSUE THREAD

**Issue by gibbster** [NYTimes/backbone.stickit#148]
Hi!

I was thinking about this problem for a while, and don't yet have a great solution. I would love to hear any tips you can provide!

On my model I have a date attribute (stored as an iso 8601 string) and a timezone attribute.

On my form I have a date input field, a time input field, and a timezone select field.

I would like my date and time field to observe both the date attribute and the timezone attribute (and use those for onGet), but I would also like the any change of date, time or timezone to update the date attribute (i.e. to be used in onSet).

In summary, changing the:
date attribute will update the date and time fields
timezone attribute will update the timezone field.
date field will update the date attribute
time field will update the date attribute
timezone field will update the date attribute and the timezone attribute.

It's a tangled mess, I know. I'm wondering if there's an elegant way of handling this with the excellent stickit plugin.

Thanks!
